src/timer/timer_game_tick.cpp

```diff
@@ -50,14 +50,20 @@ void TimeoutTimer<TimerGameTick>::Elapsed(TimerGameTick::TElapsed delta)
 	}
 }
 
+/**
+ * Every-tick handler for game tick timers.
+ * @return True iff time has progressed.
+ */
```

Comment:
No need for this comment (and the other cpp files); you already have that at the template definition in the header-file :)